2020-07-02 16:14:35,461 - C:/skydrm UI automation/venv/testCase/test_login.py[line:28] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:14:35,470 - C:/skydrm UI automation/venv/testCase/test_login.py[line:30] - DEBUG: <-----------------------------------------------test start---------------------------------------------->
2020-07-02 16:14:35,471 - C:/skydrm UI automation/venv/testCase/test_login.py[line:32] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:14:35,517 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:94] - DEBUG: User Status: Logout
2020-07-02 16:14:35,518 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:17] - DEBUG: user start login
2020-07-02 16:14:35,518 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:19] - DEBUG: user login info, username: starry.feng@nextlabs.com password: 123blue!
2020-07-02 16:14:47,792 - C:\skydrm UI automation\venv\PO\BaseAction.py[line:45] - DEBUG: no find label welcome
2020-07-02 16:14:47,792 - C:\skydrm UI automation\venv\PO\BaseAction.py[line:46] - DEBUG: Message: 

2020-07-02 16:14:47,792 - C:\skydrm UI automation\venv\PO\homePage.py[line:67] - DEBUG: local variable 'elem' referenced before assignment
2020-07-02 16:14:47,793 - C:\skydrm UI automation\venv\utility\Screen.py[line:15] - DEBUG: screen path is: ../testCase/scrren/error screen1593677687.7921908.png
2020-07-02 16:14:48,363 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:70] - DEBUG: normal account login fail
2020-07-02 16:14:48,364 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:71] - DEBUG: 
2020-07-02 16:14:48,364 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:25] - DEBUG: normal account login fail
2020-07-02 16:14:48,365 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:26] - DEBUG: 
2020-07-02 16:14:48,365 - C:/skydrm UI automation/venv/testCase/test_login.py[line:54] - DEBUG: 
2020-07-02 16:14:48,365 - C:/skydrm UI automation/venv/testCase/test_login.py[line:60] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:14:48,365 - C:/skydrm UI automation/venv/testCase/test_login.py[line:62] - DEBUG: <---------------------------------------test normal account end----------------------------------------->
2020-07-02 16:14:48,366 - C:/skydrm UI automation/venv/testCase/test_login.py[line:64] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:14:48,366 - C:/skydrm UI automation/venv/testCase/test_login.py[line:65] - DEBUG: 

2020-07-02 16:14:48,367 - C:/skydrm UI automation/venv/testCase/test_login.py[line:66] - DEBUG: 

2020-07-02 16:14:48,367 - C:/skydrm UI automation/venv/testCase/test_login.py[line:67] - DEBUG: 

2020-07-02 16:15:00,799 - C:/skydrm UI automation/venv/testCase/test_login.py[line:28] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:00,799 - C:/skydrm UI automation/venv/testCase/test_login.py[line:30] - DEBUG: <-----------------------------------------------test start---------------------------------------------->
2020-07-02 16:15:00,799 - C:/skydrm UI automation/venv/testCase/test_login.py[line:32] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:00,872 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:94] - DEBUG: User Status: Logout
2020-07-02 16:15:00,872 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:17] - DEBUG: user start login
2020-07-02 16:15:00,873 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:19] - DEBUG: user login info, username: starry.feng@nextlabs.com password: 4i234923
2020-07-02 16:15:03,348 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:62] - DEBUG: login prompt msg is: You've entered an incorrect username/password.
2020-07-02 16:15:03,349 - C:\skydrm UI automation\venv\utility\Screen.py[line:15] - DEBUG: screen path is: ../testCase/scrren/error screen1593677703.3492818.png
2020-07-02 16:15:04,101 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:67] - DEBUG: login page refresh fail
2020-07-02 16:15:04,102 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:70] - DEBUG: normal account login fail
2020-07-02 16:15:04,102 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:71] - DEBUG: 
2020-07-02 16:15:04,102 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:25] - DEBUG: normal account login fail
2020-07-02 16:15:04,102 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:26] - DEBUG: 
2020-07-02 16:15:04,102 - C:/skydrm UI automation/venv/testCase/test_login.py[line:54] - DEBUG: 
2020-07-02 16:15:04,102 - C:/skydrm UI automation/venv/testCase/test_login.py[line:60] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:04,102 - C:/skydrm UI automation/venv/testCase/test_login.py[line:62] - DEBUG: <---------------------------------------test normal account end----------------------------------------->
2020-07-02 16:15:04,103 - C:/skydrm UI automation/venv/testCase/test_login.py[line:64] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:04,103 - C:/skydrm UI automation/venv/testCase/test_login.py[line:65] - DEBUG: 

2020-07-02 16:15:04,103 - C:/skydrm UI automation/venv/testCase/test_login.py[line:66] - DEBUG: 

2020-07-02 16:15:04,103 - C:/skydrm UI automation/venv/testCase/test_login.py[line:67] - DEBUG: 

2020-07-02 16:15:20,159 - C:/skydrm UI automation/venv/testCase/test_login.py[line:28] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:20,159 - C:/skydrm UI automation/venv/testCase/test_login.py[line:30] - DEBUG: <-----------------------------------------------test start---------------------------------------------->
2020-07-02 16:15:20,160 - C:/skydrm UI automation/venv/testCase/test_login.py[line:32] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:20,231 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:94] - DEBUG: User Status: Logout
2020-07-02 16:15:20,231 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:17] - DEBUG: user start login
2020-07-02 16:15:20,232 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:19] - DEBUG: user login info, username: starry2.feng@nextlabs.com password: 123blue!
2020-07-02 16:15:22,929 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:62] - DEBUG: login prompt msg is: The account does not exist.
2020-07-02 16:15:22,930 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:65] - DEBUG: user login success
2020-07-02 16:15:22,930 - C:/skydrm UI automation/venv/testCase/test_login.py[line:60] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:22,930 - C:/skydrm UI automation/venv/testCase/test_login.py[line:62] - DEBUG: <---------------------------------------test normal account end----------------------------------------->
2020-07-02 16:15:22,930 - C:/skydrm UI automation/venv/testCase/test_login.py[line:64] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:22,931 - C:/skydrm UI automation/venv/testCase/test_login.py[line:65] - DEBUG: 

2020-07-02 16:15:22,931 - C:/skydrm UI automation/venv/testCase/test_login.py[line:66] - DEBUG: 

2020-07-02 16:15:22,931 - C:/skydrm UI automation/venv/testCase/test_login.py[line:67] - DEBUG: 

2020-07-02 16:15:36,277 - C:/skydrm UI automation/venv/testCase/test_login.py[line:28] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:36,279 - C:/skydrm UI automation/venv/testCase/test_login.py[line:30] - DEBUG: <-----------------------------------------------test start---------------------------------------------->
2020-07-02 16:15:36,280 - C:/skydrm UI automation/venv/testCase/test_login.py[line:32] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:36,281 - C:/skydrm UI automation/venv/testCase/test_login.py[line:49] - DEBUG: skip this case
2020-07-02 16:15:36,281 - C:/skydrm UI automation/venv/testCase/test_login.py[line:60] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:36,282 - C:/skydrm UI automation/venv/testCase/test_login.py[line:62] - DEBUG: <---------------------------------------test normal account end----------------------------------------->
2020-07-02 16:15:36,282 - C:/skydrm UI automation/venv/testCase/test_login.py[line:64] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:36,283 - C:/skydrm UI automation/venv/testCase/test_login.py[line:65] - DEBUG: 

2020-07-02 16:15:36,283 - C:/skydrm UI automation/venv/testCase/test_login.py[line:66] - DEBUG: 

2020-07-02 16:15:36,284 - C:/skydrm UI automation/venv/testCase/test_login.py[line:67] - DEBUG: 

2020-07-02 16:15:47,578 - C:/skydrm UI automation/venv/testCase/test_login.py[line:28] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:47,578 - C:/skydrm UI automation/venv/testCase/test_login.py[line:30] - DEBUG: <-----------------------------------------------test start---------------------------------------------->
2020-07-02 16:15:47,579 - C:/skydrm UI automation/venv/testCase/test_login.py[line:32] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:47,631 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:94] - DEBUG: User Status: Logout
2020-07-02 16:15:47,631 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:17] - DEBUG: user start login
2020-07-02 16:15:47,631 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:19] - DEBUG: user login info, username:  password: 124blue!
2020-07-02 16:15:49,382 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:62] - DEBUG: login prompt msg is: Please enter the Email Address.
2020-07-02 16:15:49,383 - C:\skydrm UI automation\venv\operation\loginOperation.py[line:65] - DEBUG: user login success
2020-07-02 16:15:49,383 - C:/skydrm UI automation/venv/testCase/test_login.py[line:60] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:49,383 - C:/skydrm UI automation/venv/testCase/test_login.py[line:62] - DEBUG: <---------------------------------------test normal account end----------------------------------------->
2020-07-02 16:15:49,383 - C:/skydrm UI automation/venv/testCase/test_login.py[line:64] - DEBUG: <------------------------------------------------------------------------------------------------------->
2020-07-02 16:15:49,384 - C:/skydrm UI automation/venv/testCase/test_login.py[line:65] - DEBUG: 

2020-07-02 16:15:49,384 - C:/skydrm UI automation/venv/testCase/test_login.py[line:66] - DEBUG: 

2020-07-02 16:15:49,384 - C:/skydrm UI automation/venv/testCase/test_login.py[line:67] - DEBUG: 

